keyboards/splitography/keymaps/beaklmu/common/keycode_functions.h

```diff
@@ -0,0 +1,660 @@
+
+// .......................................................... Keycode Primitives
+
+// register simple key press
+void tap_key(uint16_t keycode)
+{
+  register_code  (keycode);
+  unregister_code(keycode);
+}
+
+void shift_key(uint16_t keycode)
+{
+  register_code  (KC_LSFT);
+  tap_key        (keycode);
+  unregister_code(KC_LSFT);
+}
```

Comment:
Also, not needed.  `tap_code16` handles modded characters.

Eg, `tap_code16(S(keycode))` would do the same thing.